Build/Tools/NuGet/DotNetNuke.Abstractions.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>DotNetNuke.Abstractions</id>
+    <version>$version$</version>
+    <title>DNN Platform (Abstractions)</title>
+    <authors>DNN Corp</authors>
+    <owners>DNN Corp</owners>
+    <license type="expression">MIT</license>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <iconUrl>http://www.dnnsoftware.com/favicon.ico</iconUrl>
+    <projectUrl>https://github.com/dnnsoftware/Dnn.Platform</projectUrl>
+    <description>
+      Provides common abstraction API references for the DNN Platform.
+    </description>
+    <copyright>DNN and DotNetNuke are copyright 2002-2019 by DNN Corp. All Rights Reserved.</copyright>
```

Comment:
This should be adjusted. The proper copyright going forward for all items is

Copyright (c) .NET Foundation and Contributors

Reply:
Perfect! Thanks for the clarification on this

Reply:
@mitchelsellers should we create a task to update all file headers?

Reply:
I pushed changes that fixes this and updated all `.nuspec` files to use the correct copyright